### PR TITLE
docker-*: add Turkish translations

### PR DESCRIPTION
**Pull request by Yutyo**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by bl-ue** [comment]
This is beautiful @Yutyo! I'll get started reviewing this.

File: pages.tr/common/docker-network.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# docker network
+
+> Docker ağları oluştur ve yönet.
+> Daha fazla bilgi için: <https://docs.docker.com/engine/reference/commandline/network/>.
+
+- docker daemon'daki tüm müsait ve düzenlenmiş ağları sırala:
+
+`docker network ls`
+
+- Kullanıcı tarafından belirtilmiş bir ağ oluştur
```
Comment (bl-ue) [inline pages.tr/common/docker-network.md:10]:
```suggestion
- Kullanıcı tarafından belirtilmiş bir ağ oluştur:
```

File: pages.tr/common/docker-machine.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+> Docker çalıştıran makineler oluştur ve onları yönet.
+> Daha fazla bilgi için: <https://docs.docker.com/machine/reference/>.
+
+- Halihazırda çalışan docker makinelerini sırala:
+
+`docker-machine ls`
+
+- Belirli bir isim ile docker makinesi oluştur:
+
+`docker-machine create {{isim}}
```
Comment (bl-ue) [inline pages.tr/common/docker-machine.md:12]:
```suggestion
`docker-machine create {{isim}}`
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Found jJust two tiny oversights.

**Comment by Yutyo** [comment]
The change requests seem to be outdated, therefore it shouldn't say "changes requested".

**Review by bl-ue — APPROVED** [review]
It's because the person who requested the changes needs to approve, or the requested changes need to be dismissed by another collaborator.  They should be gone now.

---

Wow&mdash;really nice job @Yutyo!  This is a great batch of stuff.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**